TestVectors/employmentAuth.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@context": [
+      "https://www.w3.org/ns/credentials/v2",
+      "https://w3id.org/citizenship/v4rc1"
+    ],
+    "type": [
+      "VerifiableCredential",
+      "EmploymentAuthorizationDocumentCredential"
+    ],
+    "issuer": {
+      "id": "did:key:zDnaegE6RR3atJtHKwTRTWHsJ3kNHqFwv7n9YjTgmU7TyfU76",
+      "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQIW2NgUPr/HwADaAIhG61j/AAAAABJRU5ErkJggg=="
+    },
+    "credentialSubject": {
+      "type": [
+        "Person",
+        "EmployablePerson"
+      ],
+      "givenName": "JOHN",
+      "additionalName": "JACOB",
+      "familyName": "SMITH",
+      "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQIW2Ng+M/wHwAEAQH/7yMK/gAAAABJRU5ErkJggg==",
+      "gender": "Male",
+      "residentSince": "2015-01-01",
+      "birthCountry": "Bahamas",
+      "birthDate": "1999-07-17",
+      "employmentAuthorizationDocument": {
+        "type": "EmploymentAuthorizationDocument",
+        "identifier": "83627465",
+        "lprCategory": "C09",
+        "lprNumber": "999-999-999"
+      }
+    },
+    "name": "Employment Authorization Document",
+    "description": "Example Country Employment Authorization Document.",
```

Comment:
as above

Reply:
@TallTed so this ends up being no change. Is this what you wanted?

Reply:
> so this ends up being no change. Is this what you wanted?

I don't understand how it ends up being no change. All the earlier comments on this PR have what I believe to be editorial changes, reducing strings of 4, 5, 6 nouns to 3, 2, 1 moun, to ease comprehension by non-native English speakers and by various software tools.

Reply:
@TallTed I don't understand what the previous "as above" comment meant. I can rerun the test vector computation only if I know what to set the inputs to be. I need you and @dlongley to agree. Otherwise please recompute the test vectors that correspond to your changes and I can approve.

Reply:
I'm fine with whatever @TallTed wants as a description here.

Reply:
@Wind4Greg — The gist is "don't use a sequence of 3 or more nouns to identify something". Doing this makes comprehending and using the document more difficult for non-native English speakers (and even many native English speakers), as well as those using various accessibility tools. This is an example:
```suggestion
    "description": "Example \"Employment Authorization Document\" for Country.",
```
I'll go make specific suggestions where I had earlier said "as above".

Reply:
I'll go with `"description": "Example Employment Authorization Document"` extra escape sequences are confusing to implementers.